cps/src/main/js/workflow-editor.js

```diff
@@ -48,7 +48,34 @@ jenkinsJSModules.import('ace-editor:ace-editor-122')
                 });
 
                 editor.on('blur', function() {
-                    textarea.trigger('change');
```

Comment:
> Obviously you would delete the Groovy compilation from doCheckScript.

Ok, it was not so obvious to me - it will break compatibility. I hope nobody is relying on the existence of `doCheckScript` to request some ad-hoc script syntax validation (for some weird reason).

So, let's go that way.